src/main/resources/edu/cornell/kfs/module/cg/document/datadictionary/AwardMaintenanceDocument.xml

```diff
@@ -48,11 +48,6 @@
         </property>
         <property name="businessRulesClass"
                   value="edu.cornell.kfs.module.cg.document.validation.impl.AwardExtensionRule"/>
```

Comment:
Was this no longer needed?

Reply:
yes, I was referring to the removal of the webScriptFiles override. Thank you for the explanation.